packages/now-cli/src/util/dev/server.ts

```diff
@@ -542,7 +532,7 @@ export default class DevServer {
       configPath = getNowConfigPath(this.cwd);
       this.output.debug(`Reading ${configPath}`);
       config = JSON.parse(await fs.readFile(configPath, 'utf8'));
-      config[fileNameSymbol] = configPath;
+      config[fileNameSymbol] = basename(configPath);
```

Comment:
This should probably be a path relative to `this.cwd`, I think for the case where a `--local-config` flag is passed (so it might not be at the root level).

Reply:
Hmm, this error message is starting to become very verbose 🤔 

I found that we are already calling `basename()` for `vc deploy` so this is really a bug fix for `vc dev`.

https://github.com/vercel/vercel/blob/0986f4bcb6e6f63a756b45c153479893269052c4/packages/now-cli/src/util/config/files.ts#L131

After some thought, I think its best to use the file name here because thats what the variable is called, not file path. In the case when a user passes `--local-config`, then there error message will be directly below the path they used above so there is not reason to print the path again. I think its sufficient to be a simple file name and will look good when someone uses `vercel.prod.json` for example.

Reply:
Sounds good.